### ISSUE TITLE
npm install issue with Gatsby v4.x.x

### ISSUE THREAD

**Issue by joyfulelement** [cometkim/gatsby-plugin-linaria#140]
Hi, when I try to `npm install` (npm version: `8.1.2`) `gatsby-plugin-linaria`, with gatsby `4.4.0`, the following error occurs:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: gatsby-demo@10.0.0
npm ERR! Found: gatsby@4.4.0
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"4.4.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^2.0.0 || ^3.0.0" from gatsby-plugin-linaria@3.1.0
npm ERR! node_modules/gatsby-plugin-linaria
npm ERR!   gatsby-plugin-linaria@"3.1.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Presumably, #139 was meant to provide this support, but it seems missing the version increment for the `version` property in `package.json`?  Or perhaps the latest package yet been published to npm registry?

https://github.com/cometkim/gatsby-plugin-linaria/blob/8b9c1619292ee7643397a0d2724c839af7fe7dd3/package.json#L3

**Comment by cometkim** [comment]
Not yet, I will notice soon

**Comment by cometkim** [comment]
Fixed by @KRRISH96 at #139, published at v3.2.0

@joyfulelement :)
